.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: trusty
```

Comment:
Do they only allow certain distributions here? From looking at the logs, they are are using docker. It would be awesome if we could use our own Docker image. That would save a lot of time in the setup and build stage.

Reply:
Travis CI only allows 'precise' and 'trusty'.

A google search of "travis ci docker" yields some interesting results (https://docs.travis-ci.com/user/docker/).  I don't know enough Docker to comment on it's usability.

Reply:
Travis CI only allows 'precise' and 'trusty'.

A google search of "travis ci docker" yields some interesting results (https://docs.travis-ci.com/user/docker/). I don't know enough Docker to comment on it's usability.

Reply:
I wrote a follow-up ticket to explore this possibility